haas/ext/auth/database.py

```diff
@@ -131,6 +132,19 @@ def user_remove_project(user, project):
     db.session.commit()
 
 
+@rest_call('PATCH', '/auth/basic/user/<user>', Schema({
+    'user': basestring,
+    'is_admin': bool,
+}))
+def user_set_admin(user, is_admin):
+    get_auth_backend().require_admin()
+    user = api._must_find(User, user)
+    if user.label == os.environ.get('HAAS_USERNAME'):
```

Comment:
Ack, was this not there when I reviewed or did I just miss it?

One approach might be to allow this at the Api level, but do the sanity check in the cli. There might be a way to fish the current user out of the request, but the design of the server is going to fight you on this sort of thing -- the details of authentication are very purposefully hidden from Api calls. Since this is extension specific, this isn't an issue but: in general there may not be anything you could call a user at all.